package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A lightweight NaturalDocs-like LeafletJS-style documentation generator",
   "main": "dist/leafdoc.cjs.js",
   "browser": "dist/leafdoc.browser.js",
-  "module": "src/leafdoc.js",
+  "module": "src/leafdoc.ts",
```

Comment:
rollup is compiling from this file.

Actually, we should have Rollup compile from this file and output a non-typescript module file that this `module` field points to.